src/helpers/channels/get_stage_instance.ts

```diff
@@ -0,0 +1,21 @@
+import { cacheHandlers } from "../../cache.ts";
+import { rest } from "../../rest/rest.ts";
+import { ChannelTypes } from "../../types/channels/channel_types.ts";
+import { StageInstance } from "../../types/channels/stage_instance.ts";
```

Comment:
Use import type to import interfaces
```suggestion
import type { StageInstance } from "../../types/channels/stage_instance.ts";
```